lib/twap/util/generate_order.js

```diff
@@ -20,13 +22,22 @@ const { DUST } = Config
 const generateOrder = (state = {}, price) => {
   const { args = {}, remainingAmount } = state
   const {
-    sliceAmount, orderType, symbol, amount, lev, _margin, _futures
+    sliceAmount, orderType, symbol, amount, lev, _margin, _futures, amountDistortion
   } = args
+  let orderAmount
 
+  if (_isFinite(amountDistortion)) {
+    const plusOrMinus = Math.random() > 0.5 ? 1 : -1
+    const randomPercentage = ((Math.random() * amountDistortion * plusOrMinus) + 1) // random percentage from 1 to amountDistortion
+    orderAmount = sliceAmount * randomPercentage
```

Comment:
```
> 0.1 * 0.2
0.020000000000000004
```

can you use the bignumber lib we use also for iceberg orders?

Reply:
i used the prepareAmount from bfx-api-node-util library for it .. so thought not using bignumber would be okay .. but sure i will use bignumber for this